changebot/github/github_auth.py

```diff
@@ -111,3 +113,36 @@ def github_request_headers(installation):
     headers['Accept'] = 'application/vnd.github.machine-man-preview+json'
 
     return headers
+
+
+def get_installation_id_for_repo(repo):
```

Comment:
I am not sure what this is doing. So I'll defer this portion to @astrofrog .

Reply:
Looks good but please add a docstring :)